cypress/e2e/search.cy.ts

```diff
@@ -1,15 +1,16 @@
 import { NAVIGATION } from "../elements/navigation";
 import { SEARCH_PAGE_SELECTORS } from "../elements/search-page";
-import { SHARED } from "../elements/shared";
+import { SHARED_ELEMENTS } from "../elements/shared-elements";
 import { productsToSearch } from "../fixtures/search";
 import { navigateAndSearch } from "../support/pages/search";
 
-let typedText;
-
 describe("Search for products", () => {
+  let typedText;
```

Comment:
```suggestion
  const searchQuery;
```

Reply:
@szczecha  What do you think about that?

Reply:
IMO it should stay as 'let' because depending on the test case I use different data for this variable. 
About renaming to `searchQuery` actually fits better here, but we will need to change it also in the test body.